myceliald/src/main.rs

```diff
@@ -270,7 +270,7 @@ impl Daemon {
             _ => Reset::None,
         };
         if let Reset::State = reset_type {
-            tracing::info!("resetting state");
+            tracing::warn!("resetting state");
```

Comment:
This should be a more descriptive warning considering what it's going to the client.

Reply:
Agree, but will address this issue in different PR